doc/source/examples/protoselect_adult_cifar10.ipynb

```diff
@@ -0,0 +1,1140 @@
+{
```

Comment:
<ol><li>Fix the output warning by installing <code>IProgress</code> ?</li><li>Do we need to place <code>apply_category_mapping</code> somewhere else as the import path is confusing for this notebook (nothing to do with <code>cfrl</code>)?</li><li>If <code>cv_protoselect_euclidean</code> and <code>visualize_prototypes</code> are to be puvlic they should be exposed in <code>alibi.prototypes</code> (although let's discuss after I've seen the code as they might be specific to <code>ProtoSelect</code>?   </li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/626/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>